Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TokenizingTextBox/TokenizingTextBoxPage.xaml.cs

```diff
@@ -181,9 +178,19 @@ public void OnXamlRendered(FrameworkElement control)
             if (control.FindChildByName("ClearButton") is Button btn)
             {
                 _ttbEmailClear = btn;
-
                 _ttbEmailClear.Click += ClearButtonClick;
             }
+
+            if (_ttbEmailShowItems != null)
+            {
+                _ttbEmailShowItems.Click -= ShowButtonClick;
+            }
+
+            if (control.FindChildByName("ShowSelectedEmails") is Button showBtn)
+            {
+                _ttbEmailShowItems = showBtn;
+                _ttbEmailShowItems.Click += ShowButtonClick;
+            }
```

Comment:
We can add this as a sample command if we want: https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.SampleApp#5-for-interactive-buttons-use-samplecontrollercurrentregisternewcommand

Reply:
Same for the clear button

Reply:
I'll take a look at that in a subsequent PR as its part of the sample app. 
Merging now to get this into the toolkit release. 

thanks!